spec.md

```diff
@@ -1460,6 +1475,10 @@ example, to specify a description in a different language.
 
 ## Orphans
 
+*Deprecated. Refer to the failure cleanup sections for
```

Comment:
It's not clear exactly what deprecation means here.  Are these for reference, or are we saying this is wholly replaced by the new language?

Reply:
Can this whole Orphans section be removed? I am also unclear on which part becomes deprecated.

Reply:
Since the new language encapsulates all the cases of the orphan mitigation section, the text is irrelevant now. I didn't rip it out entirely only because I thought we'd be more comfortable giving existing spec users some time to digest the change. This way they wouldn't be surprised when they go to the bottom of the spec expecting to see the orphan mitigation section and there's nothing there.

Reply:
> be more comfortable giving existing spec users some time to digest the change

I vote remove it and add a note in the release notes to address this concern.

Reply:
I'm happy with that as well if others feel that way!

Reply:
+1 to pulling it out

Reply:
Can we say that using failure cleanups mentioned above, we could be 100% sure that orphans are removed, for e.g: considering a case when `the Platform reaches its internal retry limit.` and resource is not yet cleaned up. In such a case we have no way to clean up the resource when we are considering internal timeouts . I am okay with the change, just wanted to double check.

Reply:
I don't think there's a way we can guarantee all orphaned resources are removed in every case. This should encompass all situations where a broker is at least somewhat responsive. If a broker just stops accepting deletion requests permanently or takes an inordinate amount of time to accept deletion of a resource, though, I think it's reasonable to say the Platform is not expected to keep sending deletion requests forever in the hopes that one day it'll succeed. Presumably the Platform would let the user know that the cleanup failed, so they can out-of-band figure out what happened.